ui/src/components/Form/NumberTextField.js

```diff
@@ -0,0 +1,41 @@
+import { getIn } from 'formik'
```

Comment:
Double check if you need a number text field. 
A number text field is more of a native html feature. 
https://www.w3schools.com/tags/att_input_type_number.asp